recipes/cf-nvidia-tools/recipe.yaml

```
@@ -0,0 +1,46 @@
+context:
+  name: cf-nvidia-tools
+  version: "1.0.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - path: bin
+    target_directory: bin
+
+build:
+  skip:
+    - not linux
+  noarch: generic
+  script:
+    - ls -lah
+    - cp -r bin $PREFIX/bin
+  number: 0
+
+requirements:
+  run:
+    - __unix
+    - patchelf >=0.12
+
+tests:
+  - script:
+    - export c_stdlib_version="2.17.0"
+    - check-glibc ${CONDA_PREFIX}/lib/*.so.*
```

```suggestion
    - check-glibc "${CONDA_PREFIX}/lib/*.so.*"
```

Also would it make sense to have a default search argument or would we rather be explicit like in this test

Lastly should we add a library test dependency to ensure there is one library we can check?

---

I don't want to provide a default argument because in production, we don't want to test all of the binaries in PREFIX, just the NVIDIA redists. There is no glob that I can think of that satisfies this condition.

---

No double quotes because we want the terminal to expand the glob expression not pass a literal glob expression.